autoload/maktaba/json.vim

```diff
@@ -12,6 +12,164 @@ if !exists('maktaba#json#NULL')
       \ 'null': s:NULL, 'true': s:TRUE, 'false': s:FALSE}
 endif
 
+" Python implementation:
+
+" Initialize Vim's Python environment with the helpers we'll need.
+function! s:InitPython() abort
+  python << EOF
```

Comment:
What if you bypass `maktaba#python#ImportModule` and aggressively inline / special-case? Or is the overhead all in the file access?